models/nxos/l2_interfaces/nxos_l2_interfaces.yaml

```diff
@@ -0,0 +1,75 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+NETWORK_OS: nxos
+RESOURCE: l2_interfaces
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: nxos_l2_interfaces
+  version_added: 2.9
+  short_description: 'Manages Layer-2 Interfaces attributes of NX-OS Interfaces'
+  description: This module manages Layer-2 interfaces attributes of NX-OS Interfaces.
+  author: Trishna Guha (@trishnaguha)
+  notes:
+    - Tested against <network_os> 7.3.(0)D1(1) on VIRL
+  options:
+    config:
+      description: A dictionary of Layer-2 interface options
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+            - Full name of interface, i.e. Ethernet1/1.
+          type: str
+          required: true
+        access:
+          description:
+            - Switchport mode access command to configure the interface as a
+              Layer-2 access.
+          type: dict
+          suboptions:
+            vlan:
+              description:
+              - Configure given VLAN in access port. It's used as the access
+                VLAN ID.
+              type: int
+        trunk:
+          description:
+            - Switchport mode trunk command to configure the interface as a 
+              Layer-2 trunk.
+          type: dict
+          suboptions:
+            native_vlan:
```

Comment:
eos and junos l2 interface models have the allowed vlans first and the native vlan suboption second, while EOS does it the way you have it here. Is it worth making them all consistent?